yt/visualization/tests/test_plotwindow.py

```diff
@@ -510,6 +510,7 @@ def test_set_unit():
     assert str(slc.frb['gas', 'temperature'].units) == 'keV'
 
 WD = "WDMerger_hdf5_chk_1000/WDMerger_hdf5_chk_1000.hdf5"
+blast_wave = "amrvac/bw_2d0000.dat"
 
 @requires_ds(WD)
```

Comment:
You need to decorate this with @requires_file(blast_wave). I would also change this other `requires_ds` above to `requires_file`, it's incorrect that it's using `requires_ds` right now.

This decorator makes it so the test only runs if the file is available.

Reply:
right, my bad

Reply:
what's the correct way to stack those decorators ?

Reply:
Just put one after the other:

```
@requires_file(WD)
@requires_file(blast_wave)
def test_plot_2d():
```

Reply:
You'll of course also need to import `requires_file` from `yt.testing`.

Reply:
done ! (rebased + force pushed)